        public void exceptionOccurred(Throwable exception) {
        }

